src/allmydata/scripts/tahoe_status.py

```diff
@@ -11,21 +11,43 @@
     from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
 
 import os
-from urllib.parse import urlencode, quote as url_quote
+from urllib.parse import urlencode
 
 import json
 
 from .common import BaseOptions
 from allmydata.scripts.common import get_default_nodedir
-from allmydata.scripts.common_http import do_http, BadResponse
+from allmydata.scripts.common_http import BadResponse
 from allmydata.util.abbreviate import abbreviate_space, abbreviate_time
 from allmydata.util.encodingutil import argv_to_abspath
 
+_print = print
+def print(*args, **kwargs):
+    """
+    Builtin ``print``-alike that will even write unicode not encodeable using
+    the specified output file's encoding.
```

Comment:
It's not _entirely_ clear to me from this docstring why we need a custom `print` here, maybe just different wording could help? (I'm guessing the reason is something like, "built-in `print` will fail if it receives unicode that is not encodable using the specified output file's encoding, so we have to check all the args"...?)